substrate/client/network/common/src/sync/warp.rs

```diff
@@ -40,6 +40,29 @@ pub enum WarpSyncParams<Block: BlockT> {
 	WaitForTarget(oneshot::Receiver<<Block as BlockT>::Header>),
 }
 
+/// Warp sync configuration.
```

Comment:
I think this is in the wrong direction. Ideally we'd get rid of `sc-network-common` since I believe it was only introduced to aid in removing syncing from `sc-network`. `sc-network` should be the new common crate for all networking-related code and I'd want to see this entire crate removed in the future.

Reply:
I don't like having two similar structs differing only in the contained value either. Do you think it's generally OK to have them, just moved to `sc-network`? Or, may be, move only `WarpSyncConfig` to `sc-network`?

Reply:
Why cannot this be stored in `sc-network-sync`?

Reply:
My bad, I meant moving `WarpSyncConfig` to `sc-network-sync` while keeping `WarpSyncParams` in `sc-network-common`. But I'll try to also move `WarpSyncParams` to `sc-network-sync`.

Reply:
Done